src/app/(proper_react)/(redesign)/(authenticated)/admin/churn-subscribers/actions.tsx

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use server";
+import {
+  getAllSubscriberChurns,
+  upsertSubscriberChurns,
+} from "../../../../../../db/tables/subscriber_churns";
+import { SubscriberChurnRow } from "knex/types/tables";
+
+export async function getAllChurns() {
+  return getAllSubscriberChurns();
+}
+
+export async function upsertAllChurns(
+  churningSubscribers: SubscriberChurnRow[],
+) {
+  return upsertSubscriberChurns(churningSubscribers);
```

Comment:
I think it's part of a different PR's scope, but could we add an `updated_at` column? Or maybe even not do updates, but just insert new ones and select by the latest `created_at` date?

Just thinking of ways to recover from the inevitable faulty CSV upload, where it would be great if we could just remove the rows inserted by the faulty upload to get back to a working state.

Reply:
we could add a clear all function.

Reply:
added a red button